### ISSUE TITLE
convert checkpoints

### ISSUE THREAD

**Issue by WandernForte** [TissueImageAnalytics/CoNIC#7]
how to convert trained results .tar to available .pth?

P.S. I'm just an undergraduate of grade 2. If there is any docs I haven't considered, please point out. Thx.

**Comment by WandernForte** [comment]
why your model params is different from your competition train result model params?

**Comment by simongraham** [comment]
You should be able to directly read in the `.tar` file if you trained it with the `conic` branch in the HoVer-Net repo. Also, if you train with the `conic` branch, the model should be the same.

**Comment by WandernForte** [comment]
> You should be able to directly read in the `.tar` file if you trained it with the `conic` branch in the HoVer-Net repo. Also, if you train with the `conic` branch, the model should be the same.

Thx. I've tried change dict keys and solved this.
and here I shared part of my former error outputs.
`
Missing key(s) in state_dict: "decoder.tp.u3.0.weight", "decoder.tp.u3.1.units.0.0.weight", "decoder.tp.u3.1.units.0.0.bias", "decoder.tp.u3.1.units.0.0.running_mean", "decoder.tp.u3.1.units.0.0.running_var", ...

Unexpected key(s) in state_dict: "decoder.tp.u3.3.weight", "decoder.tp.u3.1.weight", "decoder.tp.u3.2.units.0.0.weight", "decoder.tp.u3.2.units.0.0.bias", "decoder.tp.u3.2.units.0.0.running_mean", "decoder.tp.u3.2.units.0.0.running_var", "decoder.tp.u3.2.units.0.0.num_batches_tracked", "decoder.tp.u3.2.units.0.2.weight", "decoder.tp.u3.2.units.0.3.weight", "decoder.tp.u3.2.units.0.3.bias", ...
`
